gitpod-io/gitpod
# [image-builder-mk3] Use baseserver

## Description


Use baseserver in `image-builder-mk3`. Despite our intended move of image builder to WS clusters, this is a worthwhile change to standardise the underlying server.

The second reason to do this is currently grpc server metrics are not properly registered on image-builder, moving it to base-server fixes that as a nice side-effect (we know it works due to other services).

## Related Issue(s)

Fixes #

## How to test

1. Start a workspace which requires an image build in preview
2. See image gets built

<img width="1034" alt="Screenshot 2022-09-27 at 22 06 07" src="https://user-images.githubusercontent.com/1419286/192625055-4d9b3191-0a9b-4cb4-8106-1cbb8d0ee328.png">


## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
